common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst

```diff
@@ -560,6 +560,54 @@ See `flight mode numbers here <https://github.com/ArduPilot/ardupilot/blob/maste
 - 0 = Thrust Restored
 - 1 = Thrust Loss Detected (altitude may be prioritised over yaw control)
 
+.. raw:: html
+
+   </td>
+   </tr>
+
+   <tr>
+   <td>26 = Sensor Failsafe (Sub)</td>
+   <td>
+
+- 0 = Sensor Failsafe Cleared
+- 1 = Sensor Failsafe Triggered
+
+.. raw:: html
+
+   </td>
+   </tr>
+
+   <tr>
+   <td>27 = Leak Failsafe (Sub)</td>
+   <td>
+
+- 0 = Leak Failsafe Cleared
+- 1 = Leak Detector Failsafe Triggered
+
+.. raw:: html
+
+   </td>
+   </tr>
+
+   <tr>
+   <td>28 = Pilot Input Timeout Failsafe</td>
+   <td>
+
+- 0 = Pilot Input Failsafe Cleared
```

Comment:
I had never heard of this pilot-input failsafe.  This really seems like an RC failsafe so I'm not sure why it exists.  It looks like it's sub only so it's really up to @jaxxzer .. I really worry this confuses non-sub users.

Reply:
will add sub only to it

Reply:
added